src/main/java/tech/ypsilon/bbbot/discord/command/VerifyCommand.java

```diff
@@ -52,12 +66,33 @@
                             + "have no access to your student.kit.edu mailbox you can send us a photo of your "
                             + "student id. Just make sure to censor the image, surname and matriculation number.\n"
                             + "-----\n"
```

Comment:
Maybe use """ ... """" tags

Reply:
The Project isn't compiled at Language Level 14 afaik...